doc/tutorial/get_started.rst

```diff
@@ -128,7 +128,7 @@ Complete the following steps to create the required disks in a LXD storage pool:
       | custom | remote3 |             | block        | 0       |
       +--------+---------+-------------+--------------+---------+
 
-3. Create a network
+3. Create networks
```

Comment:
This (and the following changes) should be undone now that we have a separate page for it.

Reply:
fixed